packages/core/src/components/editable-text/editable-text.md

```diff
@@ -23,34 +39,34 @@ you should center the component via flexbox or with `position` and `transform: t
 
 </div>
 
-
 @## Multiline mode
 
-By default, __EditableText__ supports _exactly one line of text_ and will grow or shrink horizontally based on the
-length of text.
-
-You may enable the `multiline` prop to use a `<textarea>` which spans multiple lines instead of a single-line
-`<input type="text">`. Multiline mode always appears at 100% width and adjusts _vertically_ based on length of text.
-Use the `minLines` and `maxLines` props to constrain the height of the component.
-
-```tsx
-<EditableText multiline={true} minLines={3} maxLines={12} {...props} />
-```
+By default, **EditableText** supports a single line of text and resizes horizontally as needed.
+Enabling the `multiline` prop transforms it into a `<textarea>`, which grows and shrinks vertically
+as content changes. Use the `minLines` and `maxLines` props to constrain the height of the component.
 
 Users may confirm text in multiline mode by pressing <kbd>Ctrl + Enter</kbd> or <kbd>Command + Enter</kbd> rather than
 <kbd>Enter</kbd>. (Pressing the <kbd>Enter</kbd> key by itself moves the cursor to the next line.) This behavior
 can be inverted with the `confirmOnEnterKey` prop.
 
-@## Usage
+@reactCodeExample EditableTextMultilineExample
 
-__EditableText__ is used like an [`<input>` element](https://facebook.github.io/react/docs/forms.html) and supports
-controlled or uncontrolled usage through the `value` or `defaultValue` props, respectively. Use `onChange` to listen to
-ongoing updates and use `onConfirm` and `onCancel` to listen only to completed or canceled edits.
+@## Intent
 
-The `onConfirm` and `onCancel` callbacks are invoked based on user interaction. The user presses <kbd>Enter</kbd>
-(or <kbd>Command + Enter</kbd> when multiline) or blurs the input to confirm the current value, or presses
-<kbd>Escape</kbd> to cancel. Canceling resets the field to the last confirmed value. Neither callback is invoked if the
-value is unchanged.
+The `intent` prop controls the visual appearance of **EditableText**, similar to
```

Comment:
We have notes on what intents mean in the button docs but not here. I don't know that we should necessarily add them here, but I think the lack of consistency is confusing. Curious if you think its worth repeating the meanings of intents in each place that we reference them. My intuition is that its worthwhile since we don't know where folks will look first.

Reply:
Yeah, that's a fair point. I'd included them at one point, but removed them while editing. I'm fine with including something for consistency though. Included in b7fbdefe8cf35e0204b322060f0bac5a753f2780